### ISSUE TITLE
OpenCloud Icon very Pale (Linux KDE Desktop Panel)

### ISSUE THREAD

**Issue by dragotin** [opencloud-eu/desktop#183]
The icon of the OpenCloud sync client appears to be very colorless and pale in the side panel of KDE.

![Image](https://github.com/user-attachments/assets/5ef6479b-43c1-4c31-9059-d3a6b88e362f)

@tbsbdr

**Comment by tbsbdr** [comment]
Violates a11y criterion https://www.w3.org/TR/WCAG22/#contrast-minimum

**Comment by TheOneRing** [comment]
Apparently, there is no proper way to detect the systray colour of KDE plasma.

Using a mask icon as we do on mac also had no effect.

**Comment by tbsbdr** [comment]
@TheOneRing should we always display it on petrol bg - woud that make sense? (not shure which variations need to be considered..)

**Comment by jnweiger** [comment]
Seen with OpenCloud Beta OpenCloud Beta 3.0.0.1834

<img width="629" height="169" alt="Image" src="https://github.com/user-attachments/assets/34d0d326-28db-4fb4-931a-ef475217773c" />


See the mouse pointer, the opencloud icon near the mouse pointer has extremely low contrast. BAD
Compare to the other icons, I have. Those that have a background *and* foreground color have much better contrast.
Second advantage: such an icon can work well on both light and dark background.

I'd suggest to use an icon that has less transparency and more background color: e.g.:

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/f718468d-e0a4-497f-9881-51b5a2a92b69" />
